### ISSUE TITLE
MP4 Video Branch Watermarking

### ISSUE THREAD

**Issue by moeiscool** [ZoneMinder/ZoneMinder#648]
The ZM i have installed is the video branch. I followed these instructions to install it. https://github.com/ZoneMinder/ZoneMinder/issues/452

I am looking to watermark my recordings. I have successfully achieved what i want on regular ZM. I just need it on the MP4 branch version of ZM. I have used ASR to search for "ffmpeg" and "mp4" but i only found anything relevant to video creation in zmvideo.pl... but i think zmvideo.pl is only for generating videos from JPEG files. I simply want to have the watermark appear on the mp4 file when it is created.

So in short after a long explanation: Where should I be looking for the commands that create the MP4 files in ZM? or do i have to add them to jpeg frames somewhere or something? sorry for the noobiness... I have spent at least 3 hours looking for this and I'm actually sweating from frustration. ANY help at all would be helpful :) Thanks in advance.

**Comment by knnniggett** [comment]
I'm closing this issue out due to inactivity.  If anyone is still actively working on this feature request, then feel free to reopen it.
